helm/templates/deployment.yaml

```diff
@@ -131,8 +131,13 @@ spec:
       priorityClassName: {{ .Values.deployment.priorityClassName }}
       {{ end -}}
       hostIPC: false
-      hostNetwork: false
       hostPID: false
+      {{ if .Values.hostNetwork }}
+      hostNetwork: true
+      dnsPolicy: ClusterFirstWithHostNet
+      {{ else }}
+      hostNetwork: false
+      {{ end }}
```

Comment:
Hi @kahirokunn , thank you for your patch! These files are generated using `code-generator`, i would suggest to open a PR against code-generator/templates files, that way we can propagate for the change for all the controllers. :+1: